src/Role/GravityForms.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace P4\MasterTheme\Role;
+
+/**
+ * Abstract Class GravityForms
+ */
+abstract class GravityForms {
+
+	/**
+	 * Add capabilities to user roles
+	 */
+	public static function add_capaibilities_to_user_roles() {
+		// We will potentially add more roles.
+		$roles        = [ 'editor' ];
+		$capabilities = [];
+
+		foreach ( $roles as $role_name ) {
+			switch ( $role_name ) {
+				case 'editor':
+					$role = get_role( $role_name );
+
+					$capabilities = [
+						'gform_full_access'           => false,
+						'gravityforms_create_form'    => true,
+						'gravityforms_create_form'    => true,
```

Comment:
I think it should be `gravityforms_delete_forms` instead of a duplicate of the previous line